documentation/hazards/Hazards.tex

```diff
@@ -128,23 +130,18 @@ \subsubsection*{Mitigation}
 \subsubsection*{Safety Requirements}
 Section 11.9 of our System Requirements Rev0 document covers this hazard.
 
-\subsection{User's Password Gets Leaked}
+\subsection{User's Magic Link Gets Leaked}
 
-\emph{Since we are no longer using a password-based system, we should rewrite this section}
-\begin{comment}
-\subsubsection*{Explanation}
-Depending on how the user stores their password (written down or in a file on the computer), an attacker may attain the user's password through various means. Ergo, the attacker may impersonate the user on Book Bazar. A worst-case-scenario would be if the user uses the same password for multiple services and the attacker could now access all these other services using the user's password.
 
+\subsubsection*{Explanation}
+To verify that a user is a member of the McMaster community, Book Bazar shall generate magic links which will be emailed to their McMaster emails. There is a risk of a malicious user either guessing the link, possibly by brute force and then use that to impersonate the user on Book Bazar platform. 
 \subsubsection*{Cause of Event}
-One of the means via which the attacker may acquire a user's password is by accessing the database where the information regarding the password is stored. This could be done either via a remote connection or physically accessing the computer. Another means of acquiring the user's password would be  via a man-in-the-middle attack, where some or all of the user's network communication is first routed to the attacker before reaching its intended destination. Lastly, an attacker may use a social engineering scheme to trick a user into providing the password to a service that may seem affiliated with Book Bazar but in-reality is completely separate.
+One of the means via which the attacker may acquire a user's magic link is by brute forcing all possible combinations. Apart from this, if the user is sharing their screen and another malicious user viewing the screen would be able to note down the magic link.
```

Comment:
...sharing their screen *then* another malicious...

Reply:
Also maybe remove "another" in this context, since there is only one malicious user?